src/libraries/pkg/test/packageTest.targets

```diff
@@ -19,6 +19,10 @@
   <Import Project="packageSettings\$(TestPackageId)\*.targets" />
   <Import Project="packageSettings\$(TestPackageId)\$(_targetFrameworkIdentifier)\*.targets" />
   <Import Project="packageSettings\$(TestPackageId)\$(TargetFramework)\*.targets" />
+  
+  <ItemGroup>
+    <IgnoredTypes Include="Microsoft.Extensions.Logging.LoggingBuilderExtensions" />
```

Comment:
We should make sure we have a comment referencing an issue about this type being duplicated.

Reply:
OK. I'll add comment linking https://github.com/dotnet/runtime/issues/33998 in the next commit